ui-v2/app/templates/dc/services/show/upstreams.hbs

```diff
@@ -0,0 +1,32 @@
+<div id="upstreams" class="tab-section">
+  <div role="tabpanel">
+  {{#if (gt gateway.Services.length 0)}}
+    <section>
+      <p>
+        Upstreams are services that may recieve traffic from this gateway. Learn more about configuring gateways in our
+        <a href={{env 'CONSUL_INGRESS_GATEWAYS_URL'}} target="_blank">documentation.</a>
```

Comment:
Just a little note here we can talk about later. I just noticed you did this for the terminating one also. If you look around the rest of the UI I'd been using:

```html
<p><a href="{{env 'CONSUL_DOCS_URL'}}/guides/acl-migrate-tokens.html" target="_blank" rel="noopener noreferrer">documentation</a>.</p>
```
Notes:

1. `noopener` and `noreferrer` `rel` attributes for all external links, including our own docs. This prevents URLs of Consul install from being tracked.
2. Super nit, can we put the fullstops outside of the link?
3. I'd been usings the same root env `CONSUL_DOCS_URL` and then hardcoding the rest of the URL to avoid having a tonne of env vars just for links. Generally a URL is used once, maybe twice tops, so its not massively worth making them into variables (not precious about that at all). I think if you do want to make env vars for every single URL we use as a link in the UI, then the name of the URL should at least include the word `DOCS` I think. If it was me though I'd stick to the above example, but again not too precious about that.